bin/codemods/src/single-tree-rendering.js

```diff
@@ -78,6 +84,39 @@ export default function transformer( file, api ) {
 	}
 
 	/**
+	 * Removes imports maintaining any comments above them
+	 *
+	 * @param {object} collection Collection containing at least one node. Comments are preserved only from first node.
+	 */
+	function removeImport( collection ) {
+		const node = collection.nodes()[ 0 ];
+
+		// Find out if import had comments above it
+		const comments = _.get( node, 'comments', [] );
+
+		// Remove import (and any comments with it)
+		collection.remove();
+
+		// Put back that removed comment (if any)
+		if ( comments.length ) {
+			const isRemovedExternal = isExternal( node );
+
+			// Find remaining external or internal dependencies and place comments above first one
```

Comment:
We might still wipe out `/** @format */` comments if there aren't any imports left where we could place comments back.

I wouldn't worry about this until we notice it's actually happening.